project/src/Level 1/SubHeader.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react'
+
+export default class SubHeader extends Component {
+  render() {
+    const { content, subHeaderClassName, subHeaderStyle } = this.props
+    return (
```

Comment:
Remember to leave one blank space between variable declarations, function itself and return statement.